File: test/fixtures/js-weak-tests/src/calc.js
export function add(a, b) {
  return a + b
}

export function classify(n) {
  if (n > 10) {
    return 'big'
  }
  if (n > 0) {
    return 'small'
  }
  return 'none'
}

export function discount(total, member) {
  if (member && total > 100) {
    return total * 0.9
  }
  return total
}

File: test/fixtures/py-weak-tests/pkg/calc.py
def add(a, b):
    return a + b


def classify(n):
    if n > 10:
        return "big"
    if n > 0:
        return "small"
    return "none"


def shipping(weight, express):
    if weight > 20:
        return 40 if express else 25
    return 15 if express else 5
